exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,23 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+//function to spell name in uppercase
+function greeting(message) {
+  return message.toUpperCase();
+}
+var greet_message = greeting("hello");
+
+function getMentorName(name) {
+  return name.toUpperCase();
+}
```

Comment:
Good work using two functions here for the different parts of the greeting - to finish the task you could write another function which brings the pieces together. E.g. shoutyGreeting("Daniel") would output "HELLO DANIEL"